website/src/css/pages/_jobs.scss

```diff
@@ -245,6 +245,42 @@
         }
     }
 
+    .reviews-carousel {
+        padding: 0 0 100px 0;
+        background-color: #fff;
+
+        .carousel-item {
+            text-align: center;
+            padding: 4%;
+            background-color: #fff;
+            height: 800px;
+
+            @media (min-width: 500px) and (max-width: 768px) {
+                height: 570px;
+                padding: 0px;
```

Comment:
Whenever you declare `0` values, you don't have to specify the unit of measurement so you can omit the `px`. Typically, prettier would automate and catch things like this for us 🙂